core/src/main/java/com/adobe/aem/commons/assetshare/components/search/impl/SearchConfigImpl.java

```diff
@@ -119,12 +121,23 @@ public String getGuessTotal() {
 
         try {
             int guessTotalAsNumber = Integer.parseInt(guessTotal);
-            return (guessTotalAsNumber < 1 || guessTotalAsNumber > MAX_GUESS_TOTAL) ? DEFAULT_GUESS_TOTAL : String.valueOf(guessTotalAsNumber);
+            if(isOverideGuessTotal() && guessTotalAsNumber > 1) {
+                return guessTotal;
+            }
+            else {
+                return (guessTotalAsNumber < 1 || guessTotalAsNumber > MAX_GUESS_TOTAL) ? DEFAULT_GUESS_TOTAL : String.valueOf(guessTotalAsNumber);
+            }
         } catch (NumberFormatException e) {
             return DEFAULT_GUESS_TOTAL;
         }
     }
 
+    @Override 
+    public boolean isOverideGuessTotal() {
```

Comment:
```suggestion
    public boolean isOverrideGuessTotal() {
```

Update the spelling?

Reply:
I updated the spelling in the Interface and Impl as well.